### PR TITLE
Reduce build time

### PR DESCRIPTION
**Pull request by jreidinger**
speeds up build about around 25%

### CODE REVIEW

**Comment by mvidner** [comment]
Yay! Once it builds ;-) please compare the RPM contents before and after so that we do not miss some corner case file.

File: package/yast2-installation.changes [hunk 1]
```diff
@@ -1,4 +1,9 @@
 -------------------------------------------------------------------
+Wed Sep 14 13:53:41 UTC 2016 - jreidinger@suse.com
+
+- reduce time needed for build package
```
Comment (mvidner) [inline package/yast2-installation.changes]: reduce time needed for building this package (bsc#MISSING)

Comment (jreidinger) [inline package/yast2-installation.changes]: no BSC and no version dump is intentionally to submit it with other change which hopefully have bsc :)

File: test/inst_disks_activate_test.rb [hunk 2]
```diff
@@ -1,16 +1,24 @@
 #!/usr/bin/env rspec
 
 require_relative "test_helper"
+
+# stub class from packager module
+module Yast
+  def self.stub_module(name)
+    class_eval "class #{name}; def self.fake_method;end;end"
```
Comment (mvidner) [inline test/inst_disks_activate_test.rb]: I tried to do this without String eval:

```rb
  def self.stub_module(name)
    c = Class.new do
      def self.fake_method; end
    end
    const_set(name, c)
  end
```

but it ends up being too cryptic. JFYI.

Comment (jreidinger) [inline test/inst_disks_activate_test.rb]: it is way how I do it :)

File: test/test_helper.rb [hunk 3]
```diff
@@ -13,48 +13,22 @@
...
-  end
-  Profile = ProfileClass.new
-
-  class ProfileLocationClass
-    def Process
-    end
-  end
-
-  ProfileLocation = ProfileLocationClass.new
+# stub class from packager module
```
Comment (lslezak) [inline test/test_helper.rb]: NP: `stub an YaST module` would be better as it's used also for some other modules (autoinstallation, country)

**Comment by jreidinger** [comment]
comparison of old and new rpm, shows that it need a bit elaboration:

```
Only in old_rpm/usr/lib/YaST2/startup/arch: i386
Only in old_rpm/usr/lib/YaST2/startup/arch: ia64
Only in old_rpm/usr/lib/YaST2/startup/arch: ppc
Only in old_rpm/usr/lib/YaST2/startup/arch: x86_64
Only in new_rpm/usr/lib/YaST2/startup: AUTHOR
Only in new_rpm/usr/lib/YaST2/startup: doc
Only in old_rpm/usr/lib/YaST2/startup/hooks: postFirstCall
Only in new_rpm/usr/lib/YaST2/startup/hooks: postFirstCall.readme
Only in old_rpm/usr/lib/YaST2/startup/hooks: postFirstStage
Only in new_rpm/usr/lib/YaST2/startup/hooks: postFirstStage.readme
Only in old_rpm/usr/lib/YaST2/startup/hooks: postSecondCall
Only in new_rpm/usr/lib/YaST2/startup/hooks: postSecondCall.readme
Only in old_rpm/usr/lib/YaST2/startup/hooks: postSecondStage
Only in new_rpm/usr/lib/YaST2/startup/hooks: postSecondStage.readme
Only in old_rpm/usr/lib/YaST2/startup/hooks: preFirstCall
Only in new_rpm/usr/lib/YaST2/startup/hooks: preFirstCall.readme
Only in old_rpm/usr/lib/YaST2/startup/hooks: preFirstStage
Only in new_rpm/usr/lib/YaST2/startup/hooks: preFirstStage.readme
Only in old_rpm/usr/lib/YaST2/startup/hooks: preSecondCall
Only in new_rpm/usr/lib/YaST2/startup/hooks: preSecondCall.readme
Only in old_rpm/usr/lib/YaST2/startup/hooks: preSecondStage
Only in new_rpm/usr/lib/YaST2/startup/hooks: preSecondStage.readme
diff -r old_rpm/usr/share/applications/YaST2/release_notes.desktop new_rpm/usr/share/applications/YaST2/release_notes.desktop
2d1
< X-SuSE-DocTeamID=ycc_release_notes
16d14
< 
Only in new_rpm/usr/share/doc/packages/yast2-installation: CONTRIBUTING.md
Only in new_rpm/usr/share/doc/packages/yast2-installation: README.md
```

**Comment by lslezak** [comment]
So far LGTM, please fix the missing files.

BTW a nice cleanup: +63 −623 lines :exclamation:  :thumbsup:

**Comment by jreidinger** [comment]
ok, now diff much better, I drop arch directory, adds readme files to hooks dir, otherwise it should be almost same.

diff:

```
Only in old_rpm/usr/lib/YaST2/startup: arch
Only in new_rpm/usr/lib/YaST2/startup: AUTHOR
diff -r old_rpm/usr/lib/YaST2/startup/First-Stage/F06-terminal new_rpm/usr/lib/YaST2/startup/First-Stage/F06-terminal
8,13d7
< # ---- DISABLED: S390 console check ----
< # . /usr/lib/YaST2/startup/arch/s390/console.sh
< # BEG S390: check if initviocons needs to be called
< # s390_check_need_initviocons
< # END S390:
< 
Only in new_rpm/usr/lib/YaST2/startup/hooks: postFirstCall.readme
Only in new_rpm/usr/lib/YaST2/startup/hooks: postFirstStage.readme
Only in new_rpm/usr/lib/YaST2/startup/hooks: postSecondCall.readme
Only in new_rpm/usr/lib/YaST2/startup/hooks: postSecondStage.readme
Only in new_rpm/usr/lib/YaST2/startup/hooks: preFirstCall.readme
Only in new_rpm/usr/lib/YaST2/startup/hooks: preFirstStage.readme
Only in new_rpm/usr/lib/YaST2/startup/hooks: preSecondCall.readme
Only in new_rpm/usr/lib/YaST2/startup/hooks: preSecondStage.readme
diff -r old_rpm/usr/lib/YaST2/startup/Second-Stage/S05-config new_rpm/usr/lib/YaST2/startup/Second-Stage/S05-config
13,17d12
< # ---- DISABLED: S390 console check ----
< # . /usr/lib/YaST2/startup/arch/s390/console.sh
< # BEG S390: check if initviocons needs to be called
< # s390_check_need_initviocons
< # END S390:
diff -r old_rpm/usr/share/applications/YaST2/release_notes.desktop new_rpm/usr/share/applications/YaST2/release_notes.desktop
2d1
< X-SuSE-DocTeamID=ycc_release_notes
16d14
< 
Only in new_rpm/usr/share/doc/packages/yast2-installation: CONTRIBUTING.md
Only in new_rpm/usr/share/doc/packages/yast2-installation: README.md
```

File: test/test_helper.rb [hunk 4]
```diff
@@ -13,48 +13,23 @@
...
-  Profile = ProfileClass.new
-
-  class ProfileLocationClass
-    def Process
-    end
-  end
-
-  ProfileLocation = ProfileLocationClass.new
+# stub module to prevent its Import
+# Useful for modules from different yast modules, to avoid build dependencies
```
Comment (lslezak) [inline test/test_helper.rb]: Overloaded *modules*, rather use something like _from different yast **packages**_.

Comment (jreidinger) [inline test/test_helper.rb]: thanks for idea, I still finding with naming different modules

**Comment by lslezak** [comment]
LGTM

**Pull request merged**